news/2020-02-19-new-featured-artist-elfensjon.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: ELFENSJóN"
+date: 2020-02-19 06:00:00 +0000
+---
+
+**ELFENSJóN** throws wide the gates to dark fantasy and Valhalla as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/69/header.jpg)
+
+Referred to us by the wonderful people of *Undead Corporation*, **ELFENSJóN** is a Japanese production group featuring a wicked fusion of dark fantasy and rock themes, all underpinned with a variety of vocal excellence.
```

Comment:
idk if it would be better to use the right capitalization for them (the same one that is used on the FA page).
```suggestion
Referred to us by the wonderful people of *UNDEAD CORPORATION*, **ELFENSJóN** is a Japanese production group featuring a wicked fusion of dark fantasy and rock themes, all underpinned with a variety of vocal excellence.
```